great_expectations/experimental/metric_repository/cloud_data_store.py

```diff
@@ -28,11 +28,23 @@ class Config:
         extra = Extra.forbid
 
 
+def orjson_dumps(v, *, default):
+    import orjson  # Import here since this is only installed in the cloud environment
+
+    # orjson.dumps returns bytes, to match standard json.dumps we need to decode
+    return orjson.dumps(
+        v,
+        default=default,
+        option=orjson.OPT_SERIALIZE_NUMPY,
+    ).decode()
+
+
 class Payload(BaseModel):
     data: PayloadData
 
     class Config:
         extra = Extra.forbid
+        json_dumps = orjson_dumps
```

Comment:
I would add loads as well

Reply:
Added! I hesitated at first since we don't yet use it but after reconsidering from your comment a user of this model would expect them both to be implemented.